lesson_13/bank/bank_app/src/main/java/com/codedifferently/lesson13/bank/BankAccountBase.java

```diff
@@ -0,0 +1,58 @@
+package com.codedifferently.lesson13.bank;
+
+import com.codedifferently.lesson13.bank.exceptions.InsufficientFundsException;
+import java.util.Set;
+
+/**
+ * represents the interface of bank accounts.
+ *
+ * @author mohamedibrahim
+ */
+public interface BankAccountBase {
```

Comment:
Why did you swap the names? Runs counter to convention I've established for the team.

IRL, there are going to be different opinions on what naming convention is best. This article gives you a glimpse: https://softwareengineering.stackexchange.com/questions/152759/should-i-add-an-abstract-prefix-to-my-abstract-classes